### ISSUE TITLE
"Use selected object as reference" doesn't affect position squares

### ISSUE THREAD

**Issue by albert-softie** [ManlyMarco/StudioExtraMoveAxis#4]
when using the position gizmo in the hud with the "Use selected object as reference" option enabled, the standard axis controls work as intended, but the square controls on the gizmo still behave as if the option was disabled, using the camera position relative to the object as reference

demonstration: https://youtu.be/fk6K1oqoO3A 
pay attention to the position values in the bottom left. notice how when using the hud gizmo, all 3 axes are affected at once, instead of just 2 as should be the case
also just ftr i've only tested this in kk, i have no idea if this also occurs in kks or any of the other games supported

**Comment by ManlyMarco** [comment]
Yes, it's a known issue, I'm not planning on fixing it since it would most likely require rewriting the gizmo code.
